crates/subspace-runtime/src/lib.rs

```diff
@@ -439,6 +443,167 @@ impl pallet_sudo::Config for Runtime {
     type WeightInfo = pallet_sudo::weights::SubstrateWeight<Runtime>;
 }
 
+pub type CouncilCollective = pallet_collective::Instance1;
+
+// TODO: update params for mainnnet
+parameter_types! {
+    // approximately 1 day for council to vote on a motion
+    pub CouncilMotionDuration: BlockNumber = 14_400;
```

Comment:
Inline math would be more readable and easier to change, same for similar values below. Also most of these if not all can be specified as inline constants, which might be more readable and require less documentation.